jetty-core/jetty-xml/src/test/java/org/eclipse/jetty/xml/XmlConfigurationTest.java

```diff
@@ -109,77 +127,80 @@ public void testPassedObject(String configure) throws Exception
         TestConfiguration.VALUE = 77;
         URL url = XmlConfigurationTest.class.getClassLoader().getResource(configure);
         assertNotNull(url);
-        XmlConfiguration configuration = new XmlConfiguration(Resource.newResource(url));
-        TestConfiguration tc = new TestConfiguration("tc");
-        configuration.getProperties().putAll(properties);
-        configuration.configure(tc);
+        try (ResourceFactory.Closeable resourceFactory = ResourceFactory.closeable())
+        {
+            XmlConfiguration configuration = new XmlConfiguration(resourceFactory.newResource(url));
+            TestConfiguration tc = new TestConfiguration("tc");
+            configuration.getProperties().putAll(properties);
+            configuration.configure(tc);
```

Comment:
Once this line is run, the try-with-resources could be finished.

Reply:
Let me change this.